cmd/nomos/migrate/configmanagement.go

```diff
@@ -40,6 +40,11 @@ func migrateConfigManagement(ctx context.Context, cc *status.ClusterClient, kube
 		printNotice("The cluster is already running as an OSS installation.")
 		return nil
 	}
+	if isHNCEnabled, err := cc.ConfigManagement.IsHNCEnabled(ctx); err != nil {
+		return err
+	} else if isHNCEnabled {
+		return fmt.Errorf("Hierarchy Controller is enabled on the ConfigManagement object. It must be disabled before migrating.")
```

Comment:
Add the migration instruction in the error message: https://cloud.google.com/kubernetes-engine/enterprise/config-sync/docs/how-to/migrate-hierarchy-controller.

Reply:
I have some concern including a hardcoded link in the output, as it's possible the link could be removed or renamed in the future. Do you see this as a concern?

Reply:
That is indeed a concern. Let's keep it as is.